pkg/sql/jobs_profiler_execution_details_test.go

```diff
@@ -220,6 +223,81 @@ func TestReadWriteProfilerExecutionDetails(t *testing.T) {
 		require.True(t, strings.Contains(string(goroutines), fmt.Sprintf("labels: {\"foo\":\"bar\", \"job\":\"IMPORT id=%d\", \"n\":\"1\"}", importJobID)))
 		require.True(t, strings.Contains(string(goroutines), "github.com/cockroachdb/cockroach/pkg/sql_test.fakeExecResumer.Resume"))
 	})
+
+	t.Run("read/write terminal trace", func(t *testing.T) {
+		jobs.RegisterConstructor(jobspb.TypeImport, func(j *jobs.Job, _ *cluster.Settings) jobs.Resumer {
+			return fakeExecResumer{
+				OnResume: func(ctx context.Context) error {
+					sp := tracing.SpanFromContext(ctx)
+					require.NotNil(t, sp)
+					sp.RecordStructured(&types.StringValue{Value: "should see this"})
+					return nil
+				},
+			}
+		}, jobs.UsesTenantCostControl)
+		var importJobID int
+		runner.QueryRow(t, `IMPORT INTO t CSV DATA ('nodelocal://1/foo') WITH DETACHED`).Scan(&importJobID)
+		jobutils.WaitForJobToSucceed(t, runner, jobspb.JobID(importJobID))
+		runner.Exec(t, `SELECT crdb_internal.request_job_execution_details($1)`, importJobID)
+		trace := checkExecutionDetails(t, s, jobspb.JobID(importJobID), "resumer-trace")
+		require.True(t, strings.Contains(string(trace), "should see this"))
```

Comment:
If you fancy, there is a.`require.Contains`

Reply:
TIL, changed